cmd/fpga_plugin/README.md

```diff
@@ -1,120 +1,290 @@
+being used, but is installed by default by the
+[FPGA plugin DaemonSet YAML](../../deployments/fpga_plugin/fpga_plugin.yaml), and is benign
+in `preprogrammed` mode.
 
-### Get source code:
-```
-    $ mkdir -p $GOPATH/src/github.com/intel/
-    $ cd $GOPATH/src/github.com/intel/
-    $ git clone https://github.com/intel/intel-device-plugins-for-kubernetes.git
-```
+If using the `preprogrammed` mode, and therefore *not* using the
+CRI-O prestart hook, runtimes other than CRI-O can be used (that is, the CRI-O hook presently
+*only* works with the CRI-O runtime).
 
-### Build FPGA device plugin:
-```
-    $ cd $GOPATH/src/github.com/intel/intel-device-plugins-for-kubernetes
-    $ make fpga_plugin
-```
+The FPGA device plugin requires a Linux Kernel FPGA driver to be installed and enabled to
+operate. The plugin supports the use of either of two drivers:
 
-### Verify kubelet socket exists in /var/lib/kubelet/device-plugins/ directory:
+- The Linux Kernel in-tree [DFL](https://www.kernel.org/doc/html/latest/fpga/dfl.html) driver
+- The out of tree [OPAE](https://opae.github.io/latest/docs/drv_arch/drv_arch.html) driver
+
+Install this component (FPGA device plugin) first, and then follow the links
+and instructions to install the other components.
+
+## Getting the source code
+
+```bash
+$ go get -d -u github.com/intel/intel-device-plugins-for-kubernetes
 ```
-    $ ls /var/lib/kubelet/device-plugins/kubelet.sock
-    /var/lib/kubelet/device-plugins/kubelet.sock
+
+## Verify node kubelet config
+
+Every node that will be running the FPGA plugin must have the
+[kubelet device-plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/)
+configured. For each node, check that the kubelet device plugin socket exists:
+
+```bash
+$ ls /var/lib/kubelet/device-plugins/kubelet.sock
+/var/lib/kubelet/device-plugins/kubelet.sock
 ```
 
-### Choose mode for FPGA device plugin
+## Deploying as a DaemonSet
 
-You can run the FPGA device plugin in either `af` or `region` mode.
+> **Note:** The DaemonSet YAML launches `af` mode by default.
 
-In `af` mode, the device plugin exposes accelerator functions
-programmed onto the device as resources. Accelerator functions identified
-by the same AFU ID are translated to resources of the same type.
+To deploy the FPGA plugin as a DaemonSet, you first need to build a container image
+for the plugin and ensure that is visible to your nodes.
 
-In `region` mode, the device plugin exposes partial reconfiguration regions
-as consumable resources. Regions with the same interface ID are
-translated to resources of the same type.
+> **Note:** The FPGA plugin [DaemonSet YAML](../../deployments/fpga_plugin/fpga_plugin.yaml)
+> also deploys the [FPGA CRI-O hook](../fpga_criohook) `initcontainer` image as well. You may
+> also wish to build that image locally before deploying the FPGA plugin to avoid deploying
+> the Docker hub default image.
 
```

Comment:
The same thing here. I'd suggest to remove this to simplify deployment instructions.